borbskeeper/ParseManager/BorbParseManager.h

```diff
@@ -32,8 +32,14 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (void)loadMoreIncompleteTasksOfUser:(NSString *)username withLaterDate:(NSDate *)date WithCompletion:(void (^)(NSMutableArray *))completion;
 
++ (void)fetchCompleteTasksOfUser:(NSString *)username WithCompletion:(void (^)(NSMutableArray *))completion;
```

Comment:
Should we add an error parameter to the completion block?
```suggestion
+ (void)fetchCompleteTasksOfUser:(NSString *)username withCompletion:(void (^)(NSMutableArray *))completion;
```